extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -25,22 +25,61 @@ define('ext.wikia.adEngine.lookup.services', [
 			rubiconFastlane,
 			prebid,
 			rubiconVulcan
-		];
+		],
+		bidIndex = {
+			'rubicon_fastlane': {
+				'pos': 0,
+				'char': 'R'
+			},
+			'ox_bidder': {
+				'pos': 1,
+				'char': 'O'
+			},
+			'amazon': {
+				'pos': 2,
+				'char': 'A'
+			},
+			'rubicon_vulcan': {
+				'pos': 3,
+				'char': 'V'
+			},
+			'prebid': {
+				'pos': 4,
+				'char': 'P'
+			}
+		},
+		bidMarker = ['x','x','x','x','x'];
+
 
 	function addParameters(providerName, slotName, slotTargeting) {
 		var params = {};
 		if (!Object.keys) {
 			return;
 		}
+
 		bidders.forEach(function (bidder) {
 			if (bidder && bidder.wasCalled()) {
 				params = bidder.getSlotParams(slotName);
 				bidder.trackState(providerName, slotName, params);
 				Object.keys(params).forEach(function (key) {
 					slotTargeting[key] = params[key];
 				});
+				if (bidder.hasResponse()) {
+					bidMarker = appendBidderMarker(bidder.getName(), bidMarker);
```

Comment:
I would rename the method, instead `append` `update`. Inside this method you are not appending anything, you are replacing/updating some value.